cupy/fft/fft.py

```diff
@@ -203,30 +252,14 @@ def _get_cufft_plan_nd(shape, fft_type, axes=None, order='C'):
         # transform over all axes
         fft_axes = tuple(range(ndim))
     else:
-        if np.isscalar(axes):
-            axes = (axes, )
```

Comment:
Note: OK to remove `axes = (axes, )` because there has been the following incompatibility.
```
>>> scipy.fftpack.fftn(numpy.ones((2, 3)), axes=0)
array([[2.+0.j, 2.+0.j, 2.+0.j],
       [0.+0.j, 0.+0.j, 0.+0.j]])
>>> cupyx.scipy.fftpack.fftn(cupy.ones((2, 3)), axes=0)
TypeError: 'int' object is not iterable
```

The behavior is different between numpy and scipy.
- `numpy.fft.fftn` rejects non-sequence `axes`.
- `scipy.fftpack.fftn` and `scipy.fft.fftn` accept non-sequence `axes`.